engine/src/flutter/lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -1156,6 +1165,9 @@ class GloballyPositionedTextEditingStrategy extends DefaultTextEditingStrategy {
       // Refocus on the elements after applying the geometry.
       focusedFormElement!.focusWithoutScroll();
       moveFocusToActiveDomElement();
+    } else {
+      placeForm();
```

Comment:
Can't this be moved out of the if-else now, since it always runs?

Reply:
@spydon sure, I'll clean that up